### DISCUSSION TITLE
Cant see DNS records link or go to records page

### DISCUSSION THREAD

**Discussion by HusseinAlkarmi** [vercel/community#6942] [Help] [answered]
### Summary

Hi guys im trying to add a record but i cant find the link in domains setting after i click edit so is there a solution for this problem or does anyone face this thing before 
Here is an example 
![Screenshot 2024-05-22 175701](https://github.com/vercel/community/assets/87301537/da7594cf-2907-4537-aaf8-522e5f6759f6)

and it should be like this 
![image](https://github.com/vercel/community/assets/87301537/43d1c858-b239-4a57-a460-2abdad9aa980)

### Steps to Reproduce

the problem is when i try to view DNS records to edit them or add a new record from the domains when i click on domain the edit

**Comment by amyegan** [comment, marked as answer]
Hey @HusseinAlkarmi. You should be able to find the list of domains for your account at [vercel.com/account/domains](https://vercel.com/account/domains). That's where you can manage settings for your custom domains that use Vercel nameservers.

**Reply by HusseinAlkarmi** [reply]
Thanks it works 👏
